tests/TestCase.php

```diff
@@ -167,4 +169,13 @@ protected function getTestIdentityTokenIncomplete(): string
 
         return $dummyPersonAuthTokenValidUntil2050;
     }
+
+    protected function getMinimalCampaign(): Campaign
+    {
+        $charity = new Charity();
+        $campaign = new Campaign();
+        $campaign->setCharity($charity);
+
+        return $campaign;
+    }
```

Comment:
If this is the minimal campaign maybe worth doing this in the campaign class? Maybe give Campaign a required but nullable $charity param? 

Ultimately I think we should do something like make the `$proxy` param of `\MatchBot\Domain\SalesforceReadProxyRepository::pull` nullable, or split that into two separate methods, one to pull a new entity and one to update existing, so that we don't need the ability to create a less-than-minimal entity. For pulling new we don't need a whole entity, we just need an SF ID.

Reply:
Created charity constructor that takes a nullable charity for now.